app/models/project.rb

```diff
@@ -242,18 +293,12 @@ def files_from_iterator(iterator_resp)
       lines
     end
 
-    def metadata_with_defaults
-      data = (metadata_json&.dup || { }).with_indifferent_access
-      Rails.configuration.project_defaults.each { |key, value| data[key] ||= value }
```

Comment:
Where do the defaults get included with the metadata in this PR? `Rails.configuration.project_defaults` pulls in the config file and inserts the default values

Reply:
I've put back the code that reads the values from the config

Reply:
Done.